stdlib/messaging/kafka/src/main/java/org/ballerinalang/messaging/kafka/nativeimpl/consumer/SubscribeWithPartitionRebalance.java

```diff
@@ -68,16 +71,19 @@
 
     public static Object subscribeWithPartitionRebalance(Strand strand, ObjectValue consumerObject, ArrayValue topics,
```

Comment:
Ideally the method should be changed to avoid getting Strand as a parameter.

Reply:
This should have been done during the kafka - Java interop task. Must have missed

Reply:
This is not changed due to we cannot pass function pointers to an inter-op function (at least then). @aashikam